client/me/purchases/billing-history/utils.jsx

```diff
@@ -157,23 +123,16 @@ export function renderTransactionQuantitySummary(
 	const isRenewal = 'recurring' === type;
 	const isUpgrade = 'new purchase' === type && new_quantity > 0;
 
-	if ( isTitanMail( product ) ) {
+	if ( isGoogleWorkspace( product ) || isTitanMail( product ) ) {
 		return renderTransactionQuantitySummaryForTitanMail(
```

Comment:
Since we now use this function for both products, it will be better to rename this function properly to a more meaningful term on inline it.

Reply:
Ah yes, very good point!